src/xbot/common/command/BaseCommandGroup.java

```diff
@@ -1,16 +1,37 @@
 package xbot.common.command;
 
+import org.apache.log4j.Logger;
+
 import com.google.inject.Inject;
 
 import edu.wpi.first.wpilibj.command.CommandGroup;
 
 public class BaseCommandGroup extends CommandGroup {
+    
+    protected Logger log;
+    
     @Inject
     SmartDashboardCommandPutter commandPutter;
     
+    public BaseCommandGroup() {
+        log = Logger.getLogger(this.getName());
+    }
+
+    public BaseCommandGroup(String name) {
+        super(name);
+        log = Logger.getLogger(this.getName());
```

Comment:
Fair point, I can see it both ways. This way enables other manipulation/overriding of what the name shakes out to be in `getName()` and that's what will appear in the smartdashboard so it would be nice if they stayed in sync. On the other hand as you said you do have to spelunk through to verify.

This is currently how things are setup in BaseCommand so in the short-run I'd vote for consistency with this PR and if we want to change it up we can modify both.